### PR TITLE
Add port Lunar Lux

### PR DESCRIPTION
**Pull request by JeodC**
# New Port for Lunar Lux

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 640x480
- [ ] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/lunarlux/Lunar Lux.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+dos2unix "$GAMEDIR/tools/Klib/GMblob.py"
+dos2unix "$GAMEDIR/tools/patchscript"
+
+# Check if patchlog.txt to skip patching
+if [ ! -f patchlog.txt ]; then
+    if [ -f "$controlfolder/utils/patcher.txt" ]; then
+        source "$controlfolder/utils/patcher.txt"
+        $ESUDO kill -9 $(pidof gptokeyb)
+    else
+        echo "This port requires the latest version of PortMaster."
```
Comment (Cebion) [inline ports/lunarlux/Lunar Lux.sh:45]:
```suggestion
        pm_message "This port requires the latest version of PortMaster."
```

File: ports/lunarlux/Lunar Lux.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    fi
+else
+    echo "Patching process already completed. Skipping."
```
Comment (Cebion) [inline ports/lunarlux/Lunar Lux.sh:48]:
```suggestion
    pm_message "Patching process already completed. Skipping."
```

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/lunarlux/Lunar Lux.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        pm_message "This port requires the latest version of PortMaster."
+    fi
+else
+    pm_message "Patching process already completed. Skipping."
+fi
+
+# Display loading splash
+if [ -f "$GAMEDIR/patchlog.txt" ]; then
+    $ESUDO ./tools/splash "splash.png" 1 
+    $ESUDO ./tools/splash "splash.png" 2000
```
Comment (cdeletre) [inline ports/lunarlux/Lunar Lux.sh:54]:
```suggestion
    $ESUDO ./tools/splash "splash.png" 2000 &
```

File: ports/lunarlux/port.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "desc_md": null,
+    "inst": "Use `download_depot 1932680 1932681 4860005663388923912` with Steam Console to download the game and add the data to `lunarlux/assets`",
+    "inst_md": null,
+    "genres": [
+      "action",
+	  "adventure",
+	  "rpg"
+    ],
+    "image": null,
+    "rtr": true,
```
Comment (cdeletre) [inline ports/lunarlux/port.json:24]:
```suggestion
    "rtr": false,
```

File: ports/lunarlux/lunarlux/tools/Klib/__pycache__/GMblob.cpython-311.pyc
Comment (cdeletre) [inline ports/lunarlux/lunarlux/tools/Klib/__pycache__/GMblob.cpython-311.pyc:1]: can be removed (will be generated on first run)

File: ports/lunarlux/lunarlux/lib/armeabi-v7a/libc++_shared.so
Comment (cdeletre) [inline ports/lunarlux/lunarlux/lib/armeabi-v7a/libc++_shared.so:1]: can be removed for this aarch64 port

File: ports/lunarlux/lunarlux/lib/armeabi-v7a/libm.so
Comment (cdeletre) [inline ports/lunarlux/lunarlux/lib/armeabi-v7a/libm.so:1]: can be removed for this aarch64 port

Comment (cdeletre) [inline ports/lunarlux/lunarlux/lib/armeabi-v7a/libm.so:1]: can be removed for this aarch64 port

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/lunarlux/Lunar Lux.sh (hunk 3 above)
Comment (JeodC) [reply ports/lunarlux/Lunar Lux.sh:54]: Not a necessary change methinks, port runs fine.

Comment (JanTrueno) [reply ports/lunarlux/Lunar Lux.sh:54]: This means the port waits for the splash to end, the idea is that it splashes while loading. But its 2 seconds so whatever ^^

**Review by JanTrueno — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

File: ports/lunarlux/Lunar Lux.sh (hunk 3 above)
Comment (cdeletre) [reply ports/lunarlux/Lunar Lux.sh:54]: Yeah I don't know what was your objective, having a splash on the screen while it's loading or showing a splash before loading the game ^^

Comment (JeodC) [reply ports/lunarlux/Lunar Lux.sh:54]: I had no idea the absence of the & was pausing the script. That does make the loading splash redundant. I'll fix.

**Review by JanTrueno — APPROVED** [review]

**Pull request merged**